frontend/javascripts/oxalis/view/right-border-tabs/tree_hierarchy_view_helpers.ts

```diff
@@ -90,6 +90,7 @@ export function insertTreesAndTransform(
       timestamp: 0,
       children: _.orderBy(treeNodeChildren, ["name"], ["asc"]),
       disableCheckbox: treeNodeChildren.length === 0,
+      expanded: group.isExpanded == null ? true : group.isExpanded,
```

Comment:
```suggestion
      expanded: group.isExpanded == null || group.isExpanded,
```

usually, if `true` or `false` appear in a logical expression, they can be removed :)